documentation/src/main/asciidoc/userguide/chapters/domain/inheritance.adoc

```diff
@@ -310,6 +310,21 @@ include::{extrasdir}/entity-inheritance-table-per-class-query-example.sql[]
 [IMPORTANT]
 ====
 Polymorphic queries require multiple UNION queries, so be aware of the performance implications of a large class hierarchy.
```

Comment:
I think the `UNION ` here should be `UNION ALL` or the following description would be kind of weird.

Reply:
There's a following description that explains the difference. This paragraph uses the UNION operator not the SQL keyword.